### ISSUE TITLE
Repository for externally created MozFest blogs and content

### ISSUE THREAD

**Issue by StevieBenton** [MozillaFoundation/Mozfest2016_production#192]
Hi everyone,

Following the comms chat today, I agreed to create a ticket for people to add links to content about MozFest that they have seen. From time to time, we can then check in and retweet or republish as required.

cc @edrushka @KevZawacki @KGorr @HPaulJohnson

**Comment by edrushka** [comment]
https://medium.com/@ggstopflat/im-heading-to-mozfest-in-london-in-a-couple-of-weeks-time-and-am-hosting-a-session-called-get-c16596f82dae#.wb28oe6er

FYI I'm not reaching out. Just posting here.

**Comment by edrushka** [comment]
http://inspirenshare.blogspot.co.uk/2016/10/learning-in-youth-zone.html

**Comment by edrushka** [comment]
https://medium.com/@BadgeList/mozfest-session-preview-using-open-badges-to-find-your-next-employee-74c20b5111d3#.h1fo4ruae

**Comment by edrushka** [comment]
https://theartsawardblog.wordpress.com/2016/10/18/mozfest-fringe-blog-takeover-day-2-2/#more-1518

**Comment by edrushka** [comment]
http://www.artsawardvoice.com/magazine/articles/interview-with-agi-k

**Comment by edrushka** [comment]
@KevZawacki how do you organize your Advocacy news roundups?

**Comment by KevZawacki** [comment]
@edrushka Not sure I follow -- as in, reports on earned media? Roundups are sent via email -- want me to fwd one?

**Comment by edrushka** [comment]
How do you collect the stories? Google alerts? Who puts them in the spreadsheet? Basically I'm just wondering what the most efficient way is to keep track of all the blogs, and indicate which ones we've invited to publish in the MozFest publication. I fear the only solution is a spreadsheet 😱

**Comment by KevZawacki** [comment]
@edrushka Yes, a combination of Google Alerts + proactive searching. [This](https://docs.google.com/spreadsheets/d/1XJgXxU6W8rzLEdAzwb0wHkYq_OFcnEEia3M-hmuQLvI/edit#gid=43979753) is what the spreadsheet looks like. 

Perhaps we simply use Medium as our "spreadsheet." Unless there's a mandate we record _all_ blogs related to MozFest, on and off Medium?

**Comment by HPaulJohnson** [comment]
I discussed this a bit with @edrushka and here's the plan:
- I'll take over monitoring social and blogs for Mozfest related content and continue to curate in the spreadsheet from here on out. 
- I will be emailing all the people who have volunteered to capture content and write about Mozfest in real-time today and they will be submitting those stories directly to me before/during/after the festival. @StevieBenton and I will evaluate the "best of" and decide on a publishing approach in real-time.
- @StevieBenton, @KevZawacki and I will meet in London on Tuesday to develop a content calendar for the weekend. This will be a rough sketch of hour-by-hour publishing plans for social and blogs. This will be fluid as the festival progresses and new content comes in, but will give us a roadmap to follow and a sense of the frequency of our communications. 

Questions or additional suggestions welcome. Just let me know.

**Comment by KevZawacki** [comment]
http://www.agik.co.uk/agi-k-mozfest2016/

**Comment by SpikeUK** [comment]
https://discourse.mozilla-community.org/t/the-magic-of-the-mozilla-festival/11652

Written by a MozFest volunteer
